zrml/swaps/src/benchmarks.rs

```diff
@@ -163,12 +172,72 @@ benchmarks! {
                 status: MarketStatus::Active,
             }
         )?;
-        let _ = T::MarketCommons::insert_market_pool(0u32.saturated_into(), 0u128);
-        let _ = bench_create_pool::<T>(caller, Some(T::MaxAssets::get().into()), None, ScoringRule::CPMM, false);
-        let _ = Pallet::<T>::mutate_pool(0, |pool| {
-            pool.pool_status = PoolStatus::Closed; Ok(())
+        let market_id = T::MarketCommons::latest_market_id()?;
+        let pool_id: PoolId = 0;
+        let _ = T::MarketCommons::insert_market_pool(market_id, pool_id);
+        let _ = bench_create_pool::<T>(
+            caller,
+            Some(a as usize),
+            None,
+            ScoringRule::CPMM,
+            false,
+            None,
+        );
+        let _ = Pallet::<T>::mutate_pool(pool_id, |pool| {
+            pool.pool_status = PoolStatus::Closed;
+            Ok(())
         });
-    }: _(RawOrigin::Root, 0u32.into(), OutcomeReport::Categorical(0))
+
+        let call = Call::<T>::admin_clean_up_pool {
+            market_id,
+            outcome_report: OutcomeReport::Categorical(0),
+        };
+    }: {
+        call.dispatch_bypass_filter(RawOrigin::Root.into())?;
+    }
+
+    admin_clean_up_pool_cpmm_scalar {
+        let caller: T::AccountId = whitelisted_caller();
+        T::MarketCommons::push_market(
+            Market {
+                creation: MarketCreation::Permissionless,
+                creator_fee: 0,
+                creator: caller.clone(),
+                market_type: MarketType::Scalar(0..=99),
+                dispute_mechanism: MarketDisputeMechanism::Authorized(caller.clone()),
+                metadata: vec![0; 50],
+                oracle: caller.clone(),
+                period: MarketPeriod::Block(0u32.into()..1u32.into()),
+                report: None,
+                resolved_outcome: None,
+                scoring_rule: ScoringRule::CPMM,
+                status: MarketStatus::Active,
+            }
+        )?;
+        let market_id = T::MarketCommons::latest_market_id()?;
```

Comment:
```suggestion
        let market_id = T::MarketCommons::push_market(
            Market {
                creation: MarketCreation::Permissionless,
                creator_fee: 0,
                creator: caller.clone(),
                market_type: MarketType::Scalar(0..=99),
                dispute_mechanism: MarketDisputeMechanism::Authorized(caller.clone()),
                metadata: vec![0; 50],
                oracle: caller.clone(),
                period: MarketPeriod::Block(0u32.into()..1u32.into()),
                report: None,
                resolved_outcome: None,
                scoring_rule: ScoringRule::CPMM,
                status: MarketStatus::Active,
            }
        )?;
```

Reply:
Mitigated in 5ab83c028b221e8df1eaefee3159e17729602796